ChangeLog.md

```diff
@@ -24,6 +24,9 @@ See docs/process.md for more on how version tagging works.
   of pthread builds so that is generated alongside the main JavaScript file.
   See #21701. ()
 - `-sASYNCIFY=2` is setting now deprecated, use `-sJSPI` instead.
+- LLVM now enables multivalue and reference-types features by default. Enabling
```

Comment:
Perhaps phrase this as something that emscripten now supports?   Maybe "Due to and upstream llvm change, the x and y features are not enabled by default in emscripten"?

Reply:
Tried doing that.